.github/workflows/unittests.yml

```diff
@@ -44,6 +58,21 @@ jobs:
           python -m pip install --upgrade cython
           python -m pip install --pre "mxnet>=2.0.0b20200802" -f https://dist.mxnet.io/python
           python -m pip install -U -e .[extras]
+      - name: Build and Install TVM
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          git clone https://github.com/apache/incubator-tvm tvm --recursive
+          cd tvm
+          mkdir -p build
+          cp cmake/config.cmake build
+          echo set\(USE_LLVM ON\) >> build/config.cmake
+          echo set\(USE_GRAPH_RUNTIME ON\) >> build/config.cmake
+          echo set\(USE_BLAS openblas\) >> build/config.cmake
```

Comment:
do we need to turn on use cuda?

Reply:
It's on CPU so there is no need to do this.